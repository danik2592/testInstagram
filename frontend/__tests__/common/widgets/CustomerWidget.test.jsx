import React from 'react';
import { createSerializer } from 'enzyme-to-json';
import { mount, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { reducer as formReducer } from 'redux-form';
import { createStore, combineReducers } from 'redux';
import { Provider, connect } from 'react-redux';
import CustomerWidget from '../../../src/common/widgets/CustomerWidget';

window.yii = {
  getCsrfToken: () => true,
};
window.advertisement = {
  contactNumber: '1',
  status: 'stepOne',
  lang: 'ru',
  allRequestId: 1,
};

// Set mock current date
Date.now = jest.fn(() => 1516854226000);

// Configuring Enzyme
configure({ adapter: new Adapter() });
expect.addSnapshotSerializer(createSerializer({ mode: 'deep' }));

const generateStore = () => createStore(combineReducers({ form: formReducer }));

describe('Testing rendering CustomerWidget', () => {
  const emptyInitialValues = {
    Customer: {
      lastName: '',
      firstName: '',
      middleName: '',
      iin: '',
      bin: '',
      address: '',
      phoneNumber: '',
      organizationName: '',
      isLegalPerson: '0',
    },
  };
  const customerInfoEmptyProps = {
    lastName: '',
    firstName: '',
    middleName: '',
    bin: '',
    iin: '',
    organizationName: '',
    homeUrl: '/ru/profile-page/admin',
    titleName: 'Заказчик',
    requestId: 1,
    titlePage: 'Данные заказчика',
    letterOfAttorney: {
      alias: 'Доверенность или документ, удостоверяющий полномочия',
      fileName: null,
      id: null,
      name: 'letterOfAttorney',
      scenario: 'pdfFormat',
      type: 'AccompanyingDocuments',
    },
    isLegalPerson: '0',
    handleDocumentChange: jest.fn(() => true),
    updateCustomer: jest.fn(() => true),
    nextPage: jest.fn(() => true),
    messages: {},
    applicant: {
      lastName: '1',
      firstName: '1',
      middleName: '1',
      iinBin: '1',
    },
    labels: {
      customer: {},
      app: {},
    },
  };
  const EmptyComponent = connect(() => ({ initialValues: emptyInitialValues }))(CustomerWidget);
  const emptyCustomer = mount(<Provider store={generateStore()}><EmptyComponent {...customerInfoEmptyProps} /></Provider>);
  it('should render with empty inputs', () => {
    expect(emptyCustomer.render()).toMatchSnapshot();
  });

  it('should render with errors in inputs', () => {
    emptyCustomer.find('.col-md-offset-10 > .btn-next').simulate('click');
    expect(customerInfoEmptyProps.updateCustomer.mock.calls.length).toBe(0);
    expect(emptyCustomer.render()).toMatchSnapshot();
  });

  const anotherCustomerInitialValues = {
    Customer: {
      lastName: 'Тест',
      firstName: 'Тест',
      middleName: 'Тест',
      iin: '050140004440',
      bin: '050140004440',
      address: 'Тест',
      phoneNumber: '7777777777',
      organizationName: 'Тест',
      isLegalPerson: '0',
    },
  };
  const anotherCustomerProps = {
    ...customerInfoEmptyProps,
    lastName: 'Тест',
    firstName: 'Тест',
    middleName: 'Тест',
    bin: '050140004440',
    iin: '050140004440',
    organizationName: 'Тест',
  };
  const AnotherCustomerComponent = connect(() => ({ initialValues: anotherCustomerInitialValues }))(CustomerWidget);
  const anotherCustomer = mount(<Provider store={generateStore()}><AnotherCustomerComponent {...anotherCustomerProps} /></Provider>);
  it('should render with anotherCustomer error', () => {
    anotherCustomer.find('.col-md-offset-10 > .btn-next').simulate('click');
    expect(anotherCustomer.find('.has-error').length).toBe(0);
    expect(anotherCustomerProps.updateCustomer.mock.calls.length).toBe(0);
    expect(anotherCustomer.render()).toMatchSnapshot();
  });

  const validCustomerProps = {
    ...anotherCustomerProps,
    letterOfAttorney: {
      alias: 'Доверенность или документ, удостоверяющий полномочия',
      fileName: 'test.pdf',
      id: 1,
      name: 'letterOfAttorney',
      scenario: 'pdfFormat',
      type: 'AccompanyingDocuments',
    },
  };
  const validCustomer = mount(<Provider store={generateStore()}><AnotherCustomerComponent {...validCustomerProps} /></Provider>);
  it('should invoke handleCustomer', () => {
    validCustomer.find('.col-md-offset-10 > .btn-next').simulate('click');
    expect(validCustomer.find('.has-error').length).toBe(0);
    expect(validCustomerProps.updateCustomer.mock.calls.length).toBe(1);
    expect(validCustomer.render()).toMatchSnapshot();
  });
});
