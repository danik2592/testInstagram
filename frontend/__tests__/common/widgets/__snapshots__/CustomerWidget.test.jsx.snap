// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing rendering CustomerWidget should invoke handleCustomer 1`] = `
<div>
  <div
    class="container new_bid"
  >
    <form>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="title-type"
          >
            Данные заказчика
          </div>
          <div
            class="information-alert"
          >
            <div
              class="information-icon"
            >
              <span
                class="image-icons point"
              />
            </div>
            <div
              class="information-text"
            >
              <div
                class="information"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="form-group"
          >
            <label
              class="radio-inline"
              title=""
            >
              <input
                checked=""
                name="Customer[isLegalPerson]"
                type="radio"
                value="0"
              />
            </label>
            <label
              class="radio-inline"
              title=""
            >
              <input
                name="Customer[isLegalPerson]"
                type="radio"
                value="1"
              />
            </label>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            id="individual"
          >
            <div
              class="row"
            >
              <div
                class="col-md-6"
              >
                <div
                  class="form-group"
                >
                  <div />
                  <span
                    class="input-group-empty"
                  >
                    <input
                      autocomplete="Customer[lastName]"
                      class="form-control"
                      id="Customer[lastName]"
                      name="Customer[lastName]"
                      type="text"
                      value="Тест"
                    />
                    <span
                      class="input-group-addon-empty"
                    />
                  </span>
                </div>
              </div>
              <div
                class="col-md-6"
              >
                <div
                  class="form-group"
                >
                  <div />
                  <span
                    class="input-group-empty"
                  >
                    <input
                      autocomplete="Customer[firstName]"
                      class="form-control"
                      id="Customer[firstName]"
                      name="Customer[firstName]"
                      type="text"
                      value="Тест"
                    />
                    <span
                      class="input-group-addon-empty"
                    />
                  </span>
                </div>
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="col-md-6"
              >
                <div
                  class="form-group"
                >
                  <div />
                  <span
                    class="input-group-empty"
                  >
                    <input
                      autocomplete="Customer[middleName]"
                      class="form-control"
                      id="Customer[middleName]"
                      name="Customer[middleName]"
                      type="text"
                      value="Тест"
                    />
                    <span
                      class="input-group-addon-empty"
                    />
                  </span>
                </div>
              </div>
              <div
                class="col-md-6"
              >
                <div
                  class="form-group"
                >
                  <input
                    class="form-control"
                    id="Customer[iin]"
                    name="Customer[iin]"
                    style="background-color: rgb(255, 255, 255);"
                    type="text"
                    value="050140004440"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-md-6"
            >
              <div
                class="form-group"
              >
                <div />
                <span
                  class="input-group-empty"
                >
                  <input
                    autocomplete="Customer[address]"
                    class="form-control"
                    id="Customer[address]"
                    name="Customer[address]"
                    type="text"
                    value="Тест"
                  />
                  <span
                    class="input-group-addon-empty"
                  />
                </span>
              </div>
            </div>
            <div
              class="col-md-6"
            >
              <div
                class="form-group"
              >
                <input
                  class="form-control"
                  id="Customer[phoneNumber]"
                  name="Customer[phoneNumber]"
                  style="background-color: rgb(255, 255, 255);"
                  type="text"
                  value="(777) 777-7777"
                />
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-md-12"
            >
              <div>
                <div
                  class="form-group"
                >
                  <label>
                    <span
                      class=""
                    >
                      Доверенность или документ, удостоверяющий полномочия
                    </span>
                  </label>
                  <div>
                    <div
                      style="font-size: 12px;"
                    />
                    <div
                      class="form-group"
                      data-test-id="dropTarget"
                      style="border: 2px dashed #aaa; padding: 10px; cursor: pointer; border-radius: 4px;"
                    >
                      <div
                        style="text-align: center; width: 100%; vertical-align: middle;"
                      >
                        <span
                          class="image-icons plus"
                        />
                      </div>
                    </div>
                    <div
                      class="row"
                    >
                      <div
                        class="col-md-12"
                        style="display: inline-flex;"
                      >
                        <input
                          class="form-control"
                          style="display: none;"
                          type="file"
                        />
                        <div
                          class="full-width"
                        >
                          <input
                            class="form-control form-control"
                            id="ormValidation"
                            readonly=""
                            style="background-color: rgb(255, 255, 255);"
                            type="text"
                            value="test.pdf"
                          />
                        </div>
                        <button
                          class="btn btn-next"
                          formnovalidate=""
                          style="text-transform: uppercase;"
                          type="button"
                        />
                        <button
                          class="btn btn-next"
                          formnovalidate=""
                          style="text-transform: uppercase;"
                          type="button"
                        />
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="from-line"
          />
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-1"
        >
          <a
            class="btn btn-exit "
            href="/ru/profile-page/admin"
            style="text-transform: uppercase;"
          />
        </div>
        <div
          class="col-md-1 col-md-offset-10"
        >
          <button
            class="btn btn-next"
            style="text-transform: uppercase;"
            type="button"
          />
        </div>
      </div>
    </form>
  </div>
</div>
`;

exports[`Testing rendering CustomerWidget should render with anotherCustomer error 1`] = `
<div>
  <div
    class="container new_bid"
  >
    <form>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="title-type"
          >
            Данные заказчика
          </div>
          <div
            class="information-alert"
          >
            <div
              class="information-icon"
            >
              <span
                class="image-icons point"
              />
            </div>
            <div
              class="information-text"
            >
              <div
                class="information"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="form-group"
          >
            <label
              class="radio-inline"
              title=""
            >
              <input
                checked=""
                name="Customer[isLegalPerson]"
                type="radio"
                value="0"
              />
            </label>
            <label
              class="radio-inline"
              title=""
            >
              <input
                name="Customer[isLegalPerson]"
                type="radio"
                value="1"
              />
            </label>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            id="individual"
          >
            <div
              class="row"
            >
              <div
                class="col-md-6"
              >
                <div
                  class="form-group"
                >
                  <div />
                  <span
                    class="input-group-empty"
                  >
                    <input
                      autocomplete="Customer[lastName]"
                      class="form-control"
                      id="Customer[lastName]"
                      name="Customer[lastName]"
                      type="text"
                      value="Тест"
                    />
                    <span
                      class="input-group-addon-empty"
                    />
                  </span>
                </div>
              </div>
              <div
                class="col-md-6"
              >
                <div
                  class="form-group"
                >
                  <div />
                  <span
                    class="input-group-empty"
                  >
                    <input
                      autocomplete="Customer[firstName]"
                      class="form-control"
                      id="Customer[firstName]"
                      name="Customer[firstName]"
                      type="text"
                      value="Тест"
                    />
                    <span
                      class="input-group-addon-empty"
                    />
                  </span>
                </div>
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="col-md-6"
              >
                <div
                  class="form-group"
                >
                  <div />
                  <span
                    class="input-group-empty"
                  >
                    <input
                      autocomplete="Customer[middleName]"
                      class="form-control"
                      id="Customer[middleName]"
                      name="Customer[middleName]"
                      type="text"
                      value="Тест"
                    />
                    <span
                      class="input-group-addon-empty"
                    />
                  </span>
                </div>
              </div>
              <div
                class="col-md-6"
              >
                <div
                  class="form-group"
                >
                  <input
                    class="form-control"
                    id="Customer[iin]"
                    name="Customer[iin]"
                    style="background-color: rgb(255, 255, 255);"
                    type="text"
                    value="050140004440"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-md-6"
            >
              <div
                class="form-group"
              >
                <div />
                <span
                  class="input-group-empty"
                >
                  <input
                    autocomplete="Customer[address]"
                    class="form-control"
                    id="Customer[address]"
                    name="Customer[address]"
                    type="text"
                    value="Тест"
                  />
                  <span
                    class="input-group-addon-empty"
                  />
                </span>
              </div>
            </div>
            <div
              class="col-md-6"
            >
              <div
                class="form-group"
              >
                <input
                  class="form-control"
                  id="Customer[phoneNumber]"
                  name="Customer[phoneNumber]"
                  style="background-color: rgb(255, 255, 255);"
                  type="text"
                  value="(777) 777-7777"
                />
              </div>
            </div>
          </div>
          <div>
            <div
              class="title-type"
            />
            <div
              class="information-alert"
            >
              <div
                class="information-icon"
              >
                <span
                  class="image-icons point"
                />
              </div>
              <div
                class="information-text"
              >
                <div
                  class="information"
                />
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-md-12"
            >
              <div>
                <div
                  class="form-group"
                >
                  <label>
                    <span
                      class=""
                    >
                      Доверенность или документ, удостоверяющий полномочия
                    </span>
                  </label>
                  <div>
                    <div
                      style="font-size: 12px;"
                    />
                    <div
                      class="form-group"
                      data-test-id="dropTarget"
                      style="border: 2px dashed #aaa; padding: 10px; cursor: pointer; border-radius: 4px;"
                    >
                      <div
                        style="text-align: center; width: 100%; vertical-align: middle;"
                      >
                        <span
                          class="image-icons plus"
                        />
                      </div>
                    </div>
                    <div
                      class="row"
                    >
                      <div
                        class="col-md-12"
                        style="display: inline-flex;"
                      >
                        <input
                          class="form-control"
                          style="display: none;"
                          type="file"
                        />
                        <div
                          class="full-width"
                        >
                          <input
                            class="form-control form-control"
                            id="ormValidation"
                            readonly=""
                            style="background-color: rgb(255, 255, 255);"
                            type="text"
                            value=""
                          />
                        </div>
                        <button
                          class="btn btn-next"
                          formnovalidate=""
                          style="text-transform: uppercase;"
                          type="button"
                        />
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="from-line"
          />
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-1"
        >
          <a
            class="btn btn-exit "
            href="/ru/profile-page/admin"
            style="text-transform: uppercase;"
          />
        </div>
        <div
          class="col-md-1 col-md-offset-10"
        >
          <button
            class="btn btn-next"
            style="text-transform: uppercase;"
            type="button"
          />
        </div>
      </div>
    </form>
  </div>
</div>
`;

exports[`Testing rendering CustomerWidget should render with empty inputs 1`] = `
<div>
  <div
    class="container new_bid"
  >
    <form>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="title-type"
          >
            Данные заказчика
          </div>
          <div
            class="information-alert"
          >
            <div
              class="information-icon"
            >
              <span
                class="image-icons point"
              />
            </div>
            <div
              class="information-text"
            >
              <div
                class="information"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="form-group"
          >
            <label
              class="radio-inline"
              title=""
            >
              <input
                checked=""
                name="Customer[isLegalPerson]"
                type="radio"
                value="0"
              />
            </label>
            <label
              class="radio-inline"
              title=""
            >
              <input
                name="Customer[isLegalPerson]"
                type="radio"
                value="1"
              />
            </label>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            id="individual"
          >
            <div
              class="row"
            >
              <div
                class="col-md-6"
              >
                <div
                  class="form-group"
                >
                  <div />
                  <span
                    class="input-group-empty"
                  >
                    <input
                      autocomplete="Customer[lastName]"
                      class="form-control"
                      id="Customer[lastName]"
                      name="Customer[lastName]"
                      type="text"
                      value=""
                    />
                    <span
                      class="input-group-addon-empty"
                    />
                  </span>
                </div>
              </div>
              <div
                class="col-md-6"
              >
                <div
                  class="form-group"
                >
                  <div />
                  <span
                    class="input-group-empty"
                  >
                    <input
                      autocomplete="Customer[firstName]"
                      class="form-control"
                      id="Customer[firstName]"
                      name="Customer[firstName]"
                      type="text"
                      value=""
                    />
                    <span
                      class="input-group-addon-empty"
                    />
                  </span>
                </div>
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="col-md-6"
              >
                <div
                  class="form-group"
                >
                  <div />
                  <span
                    class="input-group-empty"
                  >
                    <input
                      autocomplete="Customer[middleName]"
                      class="form-control"
                      id="Customer[middleName]"
                      name="Customer[middleName]"
                      type="text"
                      value=""
                    />
                    <span
                      class="input-group-addon-empty"
                    />
                  </span>
                </div>
              </div>
              <div
                class="col-md-6"
              >
                <div
                  class="form-group"
                >
                  <input
                    class="form-control"
                    id="Customer[iin]"
                    name="Customer[iin]"
                    style="background-color: rgb(255, 255, 255);"
                    type="text"
                    value=""
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-md-6"
            >
              <div
                class="form-group"
              >
                <div />
                <span
                  class="input-group-empty"
                >
                  <input
                    autocomplete="Customer[address]"
                    class="form-control"
                    id="Customer[address]"
                    name="Customer[address]"
                    type="text"
                    value=""
                  />
                  <span
                    class="input-group-addon-empty"
                  />
                </span>
              </div>
            </div>
            <div
              class="col-md-6"
            >
              <div
                class="form-group"
              >
                <input
                  class="form-control"
                  id="Customer[phoneNumber]"
                  name="Customer[phoneNumber]"
                  style="background-color: rgb(255, 255, 255);"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-md-12"
            >
              <div>
                <div
                  class="form-group"
                >
                  <label>
                    <span
                      class=""
                    >
                      Доверенность или документ, удостоверяющий полномочия
                    </span>
                  </label>
                  <div>
                    <div
                      style="font-size: 12px;"
                    />
                    <div
                      class="form-group"
                      data-test-id="dropTarget"
                      style="border: 2px dashed #aaa; padding: 10px; cursor: pointer; border-radius: 4px;"
                    >
                      <div
                        style="text-align: center; width: 100%; vertical-align: middle;"
                      >
                        <span
                          class="image-icons plus"
                        />
                      </div>
                    </div>
                    <div
                      class="row"
                    >
                      <div
                        class="col-md-12"
                        style="display: inline-flex;"
                      >
                        <input
                          class="form-control"
                          style="display: none;"
                          type="file"
                        />
                        <div
                          class="full-width"
                        >
                          <input
                            class="form-control form-control"
                            id="ormValidation"
                            readonly=""
                            style="background-color: rgb(255, 255, 255);"
                            type="text"
                            value=""
                          />
                        </div>
                        <button
                          class="btn btn-next"
                          formnovalidate=""
                          style="text-transform: uppercase;"
                          type="button"
                        />
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="from-line"
          />
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-1"
        >
          <a
            class="btn btn-exit "
            href="/ru/profile-page/admin"
            style="text-transform: uppercase;"
          />
        </div>
        <div
          class="col-md-1 col-md-offset-10"
        >
          <button
            class="btn btn-next"
            style="text-transform: uppercase;"
            type="button"
          />
        </div>
      </div>
    </form>
  </div>
</div>
`;

exports[`Testing rendering CustomerWidget should render with errors in inputs 1`] = `
<div>
  <div
    class="container new_bid"
  >
    <form>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="title-type"
          >
            Данные заказчика
          </div>
          <div
            class="information-alert"
          >
            <div
              class="information-icon"
            >
              <span
                class="image-icons point"
              />
            </div>
            <div
              class="information-text"
            >
              <div
                class="information"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="form-group"
          >
            <label
              class="radio-inline"
              title=""
            >
              <input
                checked=""
                name="Customer[isLegalPerson]"
                type="radio"
                value="0"
              />
            </label>
            <label
              class="radio-inline"
              title=""
            >
              <input
                name="Customer[isLegalPerson]"
                type="radio"
                value="1"
              />
            </label>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            id="individual"
          >
            <div
              class="row"
            >
              <div
                class="col-md-6"
              >
                <div
                  class="form-group has-error"
                >
                  <div />
                  <span
                    class="input-group-empty"
                  >
                    <input
                      autocomplete="Customer[lastName]"
                      class="form-control"
                      id="Customer[lastName]"
                      name="Customer[lastName]"
                      type="text"
                      value=""
                    />
                    <span
                      class="input-group-addon-empty"
                    />
                  </span>
                  <span
                    class="help-block"
                  >
                    undefined «undefined».
                  </span>
                </div>
              </div>
              <div
                class="col-md-6"
              >
                <div
                  class="form-group has-error"
                >
                  <div />
                  <span
                    class="input-group-empty"
                  >
                    <input
                      autocomplete="Customer[firstName]"
                      class="form-control"
                      id="Customer[firstName]"
                      name="Customer[firstName]"
                      type="text"
                      value=""
                    />
                    <span
                      class="input-group-addon-empty"
                    />
                  </span>
                  <span
                    class="help-block"
                  >
                    undefined «undefined».
                  </span>
                </div>
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="col-md-6"
              >
                <div
                  class="form-group"
                >
                  <div />
                  <span
                    class="input-group-empty"
                  >
                    <input
                      autocomplete="Customer[middleName]"
                      class="form-control"
                      id="Customer[middleName]"
                      name="Customer[middleName]"
                      type="text"
                      value=""
                    />
                    <span
                      class="input-group-addon-empty"
                    />
                  </span>
                </div>
              </div>
              <div
                class="col-md-6"
              >
                <div
                  class="form-group has-error"
                >
                  <input
                    class="form-control"
                    id="Customer[iin]"
                    name="Customer[iin]"
                    style="background-color: rgb(255, 255, 255);"
                    type="text"
                    value=""
                  />
                  <span
                    class="help-block"
                  >
                    undefined «undefined».
                  </span>
                </div>
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-md-6"
            >
              <div
                class="form-group has-error"
              >
                <div />
                <span
                  class="input-group-empty"
                >
                  <input
                    autocomplete="Customer[address]"
                    class="form-control"
                    id="Customer[address]"
                    name="Customer[address]"
                    type="text"
                    value=""
                  />
                  <span
                    class="input-group-addon-empty"
                  />
                </span>
                <span
                  class="help-block"
                >
                  undefined «undefined».
                </span>
              </div>
            </div>
            <div
              class="col-md-6"
            >
              <div
                class="form-group has-error"
              >
                <input
                  class="form-control"
                  id="Customer[phoneNumber]"
                  name="Customer[phoneNumber]"
                  style="background-color: rgb(255, 255, 255);"
                  type="text"
                  value=""
                />
                <span
                  class="help-block"
                >
                  undefined «undefined».
                </span>
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-md-12"
            >
              <div>
                <div
                  class="form-group"
                >
                  <label>
                    <span
                      class=""
                    >
                      Доверенность или документ, удостоверяющий полномочия
                    </span>
                  </label>
                  <div>
                    <div
                      style="font-size: 12px;"
                    />
                    <div
                      class="form-group"
                      data-test-id="dropTarget"
                      style="border: 2px dashed #aaa; padding: 10px; cursor: pointer; border-radius: 4px;"
                    >
                      <div
                        style="text-align: center; width: 100%; vertical-align: middle;"
                      >
                        <span
                          class="image-icons plus"
                        />
                      </div>
                    </div>
                    <div
                      class="row"
                    >
                      <div
                        class="col-md-12"
                        style="display: inline-flex;"
                      >
                        <input
                          class="form-control"
                          style="display: none;"
                          type="file"
                        />
                        <div
                          class="full-width"
                        >
                          <input
                            class="form-control form-control"
                            id="ormValidation"
                            readonly=""
                            style="background-color: rgb(255, 255, 255);"
                            type="text"
                            value=""
                          />
                        </div>
                        <button
                          class="btn btn-next"
                          formnovalidate=""
                          style="text-transform: uppercase;"
                          type="button"
                        />
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="from-line"
          />
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-1"
        >
          <a
            class="btn btn-exit "
            href="/ru/profile-page/admin"
            style="text-transform: uppercase;"
          />
        </div>
        <div
          class="col-md-1 col-md-offset-10"
        >
          <button
            class="btn btn-next"
            style="text-transform: uppercase;"
            type="button"
          />
        </div>
      </div>
    </form>
  </div>
</div>
`;
